flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBIncrementalCheckpointUtilsTest.java

```diff
@@ -99,7 +99,7 @@ public void testChooseTheBestStateHandleForInitial() {
         Assert.assertNull(
                 RocksDBIncrementalCheckpointUtils.chooseTheBestStateHandleForInitial(
                         keyedStateHandles,
-                        new KeyGroupRange(3, 5),
+                        new KeyGroupRange(13, 15),
```

Comment:
The previous test is want to test that with `0.75` threshold, we might select the init handle, however, this is not true if changed the default value to `0.0`.